src/libs/Notification/PushNotification/index.native.ts

```diff
@@ -27,14 +28,7 @@ const notificationEventActionMap: NotificationEventActionMap = {};
  */
 function pushNotificationEventCallback(eventType: EventType, notification: PushPayload) {
     const actionMap = notificationEventActionMap[eventType] ?? {};
-    let payload = notification.extras.payload;
-
-    // On Android, some notification payloads are sent as a JSON string rather than an object
-    if (typeof payload === 'string') {
-        payload = JSON.parse(payload);
-    }
-
-    const data = payload as PushNotificationData;
+    const data = getValidPushNotificationData(notification);
```

Comment:
```suggestion
    const data = getPushNotificationData(notification);
```

Keep that naming consistent everywhere